File: ios/objc/StorylyDemo/StorylyDemo/IntegrationDemo/UICustomizationViewController.m
//
//  UICustomizationViewController.m
//  StorylyDemo
//
//  Created by Haldun Melih Fadillioglu on 7.02.2022.
//  Copyright © 2022 App Samurai Inc. All rights reserved.
//

#import "UICustomizationViewController.h"

@interface UICustomizationViewController ()

@property (weak, nonatomic) IBOutlet StorylyView *storylyView;

@end

@implementation UICustomizationViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
//    self.storylyView.storylyInit = [[StorylyInit alloc] initWithStorylyId:  @"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhY2NfaWQiOjU1NiwiYXBwX2lkIjoxMzg5LCJpbnNfaWQiOjEwNDA5fQ.kXqBdpUcKaJe7eA98PqHahMDf-123Uhb82t_mYzbBUM"
//        ];
    self.storylyView.rootViewController = self;
    self.storylyView.delegate = self;
}



@end

File: ios/swift/StorylyDemo/StorylyDemo/IntegrationDemo/UICustomizationViewController.swift
//
//  UICustomizationViewController.swift
//  StorylyDemo
//
//  Created by Haldun Melih Fadillioglu on 5.02.2022.
//  Copyright © 2022 App Samurai Inc. All rights reserved.
//

import UIKit
import Storyly

class UICustomizationViewController: UIViewController {
    
    @IBOutlet weak var storylyView: StorylyView!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        storylyView.rootViewController = self
    }
}
